backend/apiServer/model/file.go

```diff
@@ -0,0 +1,36 @@
+//Package model refers to model part of mvc.
```

Comment:
Should probably be merged with FileModel struct and rest in a struct like CodeSnipetModel or somthing to show why you got StartLine and EndLine as all "Files" start on line 0 and end on EOF.

Reply:
Agreed with part that the strcut should be renamed to CodeSnippetModel but i dont agree with that it should be merged with fileModel because the functionality doesn't really require any data from fileModel. The functionality should be part of CodeSnippetModel.

Reply:
Thats fine, as long as we are consistent.